app/main.py

```diff
@@ -12,36 +12,35 @@ def print_name(self) -> None:
         print(f"Hello, I'm {self.name}")
 
     def feed(self) -> int:
-        if self.is_hungry is True:
-            print(f"Eating {self.appetite} food points...")
-            self.is_hungry = False
-            return self.appetite
-        return 0
+        if self.is_hungry is False:
```

Comment:
You previous code was ok, I only suggest to do condition simplier like ```if self.is_hungry:```

Reply:
Ok, I guess got it wrong - fixed.